apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -1051,12 +1051,25 @@ internal BigtableClientImpl(BigtableServiceApiClient[] clients, string appProfil
         public override BigtableClient WithAppProfileId(string appProfileId) =>
             new BigtableClientImpl(_clients, appProfileId);
 
+        internal IClock Clock => UnderlyingClientSettings?.Clock ?? SystemClock.Instance;
```

Comment:
It might be worth copying this out of the settings on construction. I believe that's what we do for ApiCall - changes to settings are irrelevant after client construction. It would be good to be consistent about that.

Reply:
Done